test/extractor/CaretValueRuleExtractor.test.ts

```diff
@@ -406,6 +416,58 @@ describe('CaretValueRuleExtractor', () => {
     });
   });
 
+  describe('ValueSet', () => {
+    it('should not extract any ValueSet caret rules when only keyword-based properties have changed', () => {
+      const caretRules = CaretValueRuleExtractor.processResource(
+        looseVS,
+        defs,
+        looseVS.resourceType
+      );
+      expect(caretRules).toEqual<ExportableCaretValueRule[]>([]);
+    });
+
+    it('should extract ValueSet caret rules when non keyword-based properties have changed', () => {
+      const vs = cloneDeep(looseVS);
+      vs.status = 'active';
+      vs.compose.inactive = true;
+      const caretRules = CaretValueRuleExtractor.processResource(vs, defs, vs.resourceType);
```

Comment:
To ensure that _only_ the expected values are received, we should assert that the length is 2:
```ts
expect(caretRules).toHaveLength(2);
```

Or we should just do a full compare on the array instead of the separate assertions on lines 440 and 441:
```ts
expect(caretRules).toEqual([expectedRule1, expectedRule2]);
```

Reply:
BTW -- I modified this test to also set `identifier` to a value and confirmed that with the current code, it does _not_ export a caret rule for `identifier` -- so we do need to fix that comparison in `processResource` and probably go ahead and use `identifier` in the tests as well.

Reply:
Fixed in https://github.com/FHIR/GoFSH/pull/30/commits/27fd64dd2479cd55d7eb17f02657e9d1e00f40fd